### DISCUSSION TITLE
[BP#35] Eviter d'utiliser des images bitmap pour l'interface

### DISCUSSION THREAD

**Discussion by devcrafting** [cnumr/best-practices#205] [Bonnes pratiques] [answered]
Cette BP me semble très bien, par contre j'ai du mal avec l'exemple, l'image ne m'inspire rien du tout 🤔, on doit pouvoir faire la même chose avec une image plus courante utilisée dans les interfaces.

Le terme bitmap a porté à confusion dans la BP 80 (https://github.com/cnumr/best-practices/discussions/33), s'inspirer des discussions de celle-ci pour clarifier?

**Comment by ACTLEM** [comment]
:-D oui sûrement que l'on peut utiliser une autre image. 

En phase pour clarifier le terme bitmap.

**Comment by ACTLEM** [comment, marked as answer]
J'ai créé l'issue pour modification: https://github.com/cnumr/best-practices/issues/292
